### PR TITLE
metadata orchestrator: set top-level lanugage field from tags

### PR DESCRIPTION
**Pull request by erohmensing**
## What

Close https://github.com/airbytehq/airbyte-internal-issues/issues/9358

Set a top-level `language` key on the registry entry so that we can smoothly [consume this information in the platform](https://github.com/airbytehq/airbyte-internal-issues/issues/9247). This is a temporary solution - we should [migrate this info to top-level in the metadata file itself](https://github.com/airbytehq/airbyte-internal-issues/issues/9363)

## Can this PR be safely reverted and rolled back?

- [x] YES 💚
- [ ] NO ❌

### CODE REVIEW

**Comment by erohmensing** [comment]
* **#44466** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/44466?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @erohmensing and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by erohmensing** [comment]
@bnchrch If we like this, once we merge it, can you help me run a backfill of `generate_registry_entry` for the `latest` metadata files?

File: airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/registry_entry.py [hunk 1]
```diff
@@ -246,6 +266,9 @@ def metadata_to_registry_entry(metadata_entry: LatestMetadataEntry, override_reg
     # Add Dependency information
     overridden_metadata_data["packageInfo"] = apply_package_info_fields(overridden_metadata_data, metadata_entry)
 
+    # Add language field
+    overridden_metadata_data = apply_language_field(overridden_metadata_data)
```
Comment (bnchrch) [inline airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/registry_entry.py:270]: This is great but we should update our registry type as well 
https://github.com/airbytehq/airbyte/blob/e519f0fc9674afe8af74ee2a8d4e8f9dc05c411e/airbyte-ci/connectors/metadata_service/lib/metadata_service/models/generated/ConnectorRegistryV0.py

File: airbyte-ci/connectors/metadata_service/orchestrator/tests/test_registry.py [hunk 2]
```diff
@@ -443,3 +443,44 @@ def test_icon_url():
...
+    }
+
+    mock_metadata_entry = mock.Mock()
+    mock_metadata_entry.metadata_definition.dict.return_value = metadata
+
+    result = metadata_to_registry_entry(mock_metadata_entry, "oss")
+    assert result["language"] == "manifest-only"
+
+
+def test_language_from_tags_does_not_override_top_level_language():
```
Comment (bnchrch) [inline airbyte-ci/connectors/metadata_service/orchestrator/tests/test_registry.py:468]: <3

**Review by bnchrch — CHANGES_REQUESTED** [review]

File: airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/registry_entry.py (hunk 1 above)
Comment (erohmensing) [reply airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/registry_entry.py:270]: Ah yes, thank you!!

File: airbyte-ci/connectors/metadata_service/orchestrator/pyproject.toml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "orchestrator"
-version = "0.3.0"
+version = "0.4.0"
```
Comment (bnchrch) [inline airbyte-ci/connectors/metadata_service/orchestrator/pyproject.toml:3]: ❗ Not really required, but good hygiene. You should run `poetry update metadata_service` from the orchestrator package to update the lock file.

**Review by bnchrch — APPROVED** [review]
a-woman-sitting-at-a-table-with-a-netflix-logo-on-the-bottom

While she might not be a fan of this language, I am. Approved!

**Pull request merged**